test/unit-tests/index.test.ts

```diff
@@ -1414,7 +1417,7 @@ describe("Custom Domain Plugin", () => {
         });
 
         it("Domain summary failed", async () => {
-            AWS.mock("APIGateway", "getDomainName", (params, callback) => {
+            AWS.mock("ApiGatewayV2", "getDomainName", (params, callback) => {
```

Comment:
I don't know why this was set to APIGateway to begin with.  The actual code uses the ApiGatewayV2 to make the call to getDomainName.  Tests worked fine before my changes ( I don't know why ), but after they would hang on this test until I updated the mock to ApiGatewayV2 to match the actual code.